src/cascadia/TerminalSettingsModel/Profile.idl

```diff
@@ -31,7 +31,8 @@ namespace Microsoft.Terminal.Settings.Model
     enum BellStyle
     {
         Audible = 0x1,
-        Visual = 0x2,

```

Comment:
If you want to fake deprecate "visual", you'll probably want to keep this.

Reply:
When we ask the SettingsModel/Profile for the bellstyle, we don't want it to say `Visual`, we want it to say `Window | Taskbar`, so I don't think we need this here anymore